### DISCUSSION TITLE
Possible to *add* menu items or tiles?

### DISCUSSION THREAD

**Discussion by seidler2547** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#636] [Q&A] [answered]
I recently got a Xiaomi X10+ and this card works great with it. However, I would like to add the mode (sweeping/mopping/mopping+sweeping) and the mop pad humidity to the menu. The only way I could see how to do this was to generate the static config, then add the items I wanted to add. But that seems somehow overcomplicated and not future-proof. So I was wondering if there was a way to keep the existing auto-generated config and just add menu items and tiles?

**Comment by PiotrMachowski** [comment]
Yes, UI editor provides just basic functionalities, you can do much more in yaml configuration.
* [Icons](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card#icon-list-entry-options)
* [Tiles](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card#tile-list-entry-options)
* [Map modes](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card#map-modes-options)

**Reply by seidler2547** [reply]
That wasn't my question. The question is whether there is a way to **add** items instead of replacing all items.

**Reply by PiotrMachowski** [reply, marked as answer]
Yes, using `append_icons: true` and `append_tiles: true`
